### ISSUE TITLE
Using rest props as element attributes

### ISSUE THREAD

**Issue by Vovan-VE** [dazld/react-on-visible#29]
Hello.

I use `<OnVisible/>` to render comments tree for a forum topic. Each single comment is an `<OnVisible/>` instance. I wish to add, for example, an `id` attribute to the element, but there are no ability for that. Now I just add extra wrapper element only to add `id` to it, but this is +1 nested elements on each reply level. I need an `id` attribute to whole element containing a comment, not to any part of a comment.

```js
<OnVisible
    onChange={handler}
    wrappingElement="article"
    className="..."
    id={`comment-${id}`}
/>
```

Someone else would to add some other attributes like `data-*` or `title`. Someone else would to use `a` or `img` element directly without an extra wrapper element.

```js
<OnVisible wrappingElement="a" href="..." title="..."/>
<OnVisible wrappingElement="img" src="..." alt="..."/>
```

There are a common practice to use `...rest` props for root element's attributes.

**Comment by dazld** [comment]
that seems like a good enhancement - if you'd like to have a go at a PR, that would be great!

**Comment by dazld** [comment]
Fixed in #30
